easybuild/easyblocks/a/advisor.py

```diff
@@ -0,0 +1,41 @@
+"""
+EasyBuild support for installing the Intel Advisor XE, implemented as an easyblock
+
+@author: Lumir Jasiok (IT4Innovations)
+"""
+
+from easybuild.easyblocks.generic.intelbase import IntelBase
+
+class EB_Advisor(IntelBase):
+    """
+    Support for installing Intel Advisor XE
+    """
+
+    def sanity_check_step(self):
+        """Custom sanity check paths for Advisor"""
+
+        custom_paths = {
+            'files': ["advisor_xe/lib64/libadvixe_runtool_6.6.so", "advisor_xe/lib64/libadvixe_dvt_core_6.5.so"],
```

Comment:
where do these versions come from?

maybe just check for a non-empty `advisor_xe/lib64`, which you're already doing below? so leave this list empty (`[]`)?